sympy/ntheory/tests/test_modular.py

```diff
@@ -32,3 +32,9 @@ def test_modular():
     assert solve_congruence(*list(zip((1, 1, 2), (3, 2, 4)))) is None
     raises(
         ValueError, lambda: solve_congruence(*list(zip([3, 4, 2], [12.1, 35, 17]))))
+    assert crt_cartesian([[3, 5], [3, 7]], [7, 11]) == [3, 73, 47, 40]
```

Comment:
A doctest/test like `assert crt_cartesian([[1,2],[4]],[3,5]) == [4, 14]` might be useful to show that the length of the remainders does not need to be the same.